modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -38,7 +48,10 @@
   $disable_gc_alerts = false,
   $manage_repo = true,
   $open_firewall_from_all = false,
-  $backup_enabled = false
+  $backup_enabled = false,
+  $cors_enabled = false,
+  $cors_allow_headers = 'X-Requested-With, Content-Type, Content-Length',
+  $cors_allow_origin = 'http://app.quepid.com'
```

Comment:
Just checking my own understanding: are we adding the quepid config to the defaults so that we can start using quepid in production by just flipping the `cors_enabled` flag (or adding it to production hiera)?